seabird.proto

```diff
@@ -27,6 +27,29 @@ message Event {
     PerformActionEvent perform_action = 9;
     PerformPrivateActionEvent perform_private_action = 10;
   }
+
+  // Tags are a set of key-value items which may expose additional information
+  // about the given Event.
+  //
+  // All tags SHOULD be prefixed with something unique to that plugin or
+  // backend. All tags starting with `seabird/` are reserved for use by
+  // seabird-core.
+  //
+  // Currently known tags:
+  // - proxy/skip - this can be set to '1' if the message should not be
+  //   proxied. Any other value means the message should be proxied. As an
+  //   example, this is useful for metadata which some backends may already
+  //   provide. Generally if a plugin uses this, it should respond to both
+  //   normal events and the "send" variant of that message.
```

Comment:
Can you link the specific proto you're talking about in this comment?

Reply:
The challenge is that it refers to a number of protos. MessageEvent/SendMessageEvent, PrivateMessageEvent/SendPrivateMessageEvent, ActionEvent/PerformActionEvent, etc.

Reply:
Can you give at least an example or two then?